src/linkedlist.js

```diff
@@ -0,0 +1,148 @@
+'use strict'
+
+class Node {
+
+  constructor( data ) {
+    this.data = data
+    this.next = null
+  }
+
+}
+export default class LinkedList {
+
+  constructor () {
+      this.head = null
+      this.tail = null
+      this.count = 0
+
+
+  }
+  // Inserts a node (with the provided data) to the head of the list
+  insertFirst ( value ) {
+    //create new node
+    const node = new Node( value )
+    //find if head is defined
+    if ( this.head ) {
+      //set old head to next of new node
+      node.next = this.head
+      //if head is defined, set new node to head
+    } else {
+      //if not defined set current node to head
+      this.tail = node
+    }
+    this.head = node
+    this.count++
+  }
+  // Returns the size of the list (number of nodes)
+  size () {
+    return this.count
+  }
+
+  // Inserts a node (with the provided data) to the tail of the list
+  insert ( value ) {
+    //create new node
+    const node = new Node( value )
+    //find if tail is defined
+    if ( this.head === null ) {
+    //set oldtail next to new node
+      this.head = node
+      this.tail = node
```

Comment:
You can extract this line to outside of the conditionals since you're doing it in both the `if` and `else` conditions.